buf.yaml

```diff
@@ -415,7 +415,8 @@ lint:
     - external/habitat/event.proto # this file is owned by habitat and copied here
 breaking:
   use:
-    - PACKAGE
+    # TODO: TEMPORARY CHANGE JUST FOR THIS PR!
+    - FILE
```

Comment:
Should this be reverted before this PR gets merged to master?

Reply:
when we rebase from master we got this changes

Reply:
I can see this change in master too. Not sure why we have these changes in this PR too as rebase should not introduce new changes to the PR. But this clears the confusion for me. We can check why it was introduced in the master as it looks like a change meant for a PR.